Windows build exception from chocolately/powershell

https://github.com/natcap/invest/actions/runs/5227264154/jobs/9476490342?pr=1322
```
Chocolatey installed 2/2 packages. 
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
RefreshEnv.cmd does not work when run from this process. If you're in PowerShell, please 'Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1' and try again.
Error: Process completed with exit code 1.
```

I'll try just doing exactly what the error message suggests

--- a/.github/workflows/build-and-test.yml
+++ b/.github/workflows/build-and-test.yml
@@ -310,6 +310,7 @@ jobs:
         run: |
           choco install zip unzip
           $env:PATH += ";C:\ProgramData\chocolatey\bin"
+          Import-Module $env:ChocolateyInstall\helpers\chocolateyProfile.psm1
           refreshenv  # Choco-provided command to reload environment variables
 
       - name: Build userguide

